### DISCUSSION TITLE
Gibt es eine Möglichkeit die Radien der Auswahlliste bei Werkzeuge/Umgebung zu ergänzen?

### DISCUSSION THREAD

**Discussion by faulgurke** [e-netze/webgis-community#33] [Q&A]
Beim Werkzeug/Umgebungskreis gibt es eine Auswahlliste von 100 m bis 500 km. Ist es möglich, diese Auswahlliste zu ergänzen?

**Comment by gis-eni** [comment]
* Variante 1:
  Geh im CMS auf ``KartenViewer`` => ``Werkzeuge``. Im Eingabefeld unter Umgebungskreis, gib deine gewünschten Radien als Array an:
  ``[10,20,30,50,100,200,500]``.

* Variante 2: (ab Build 5.23.901) Empfohlen
  in der ``api.config`` eine Section für das Werkzeug anlegen:

``` javascript
<section name="tool-measure-circle">
    <add key="radii" value="5,10,25" />
</section>
``` 
Hier die Radien (nur ganze Meter) mit Beistrich getrennt.
Werden im CMS und in der ``api.config`` Radien parametriert, ist das Ergebnis die Verinigungsmenge beider.
